test/microapps-ui-xcm/microapps-ui-main.deposit.test.ts

```diff
@@ -134,11 +134,11 @@ describe("Microapps UI deposit modal tests - no action", () => {
     await depositModal.selectToken(KSM_ASSET_NAME);
     await depositModal.enterValue("1");
 
-    // await depositModal.waitForContinueState(true, 180000);
+    await depositModal.waitForContinueState(true, 180000);
```

Comment:
:O 180s?

Reply:
ah yea leftover from debug, not need to be that long , will fix!